src/main/java/seedu/algobase/logic/commands/ImportCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.algobase.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.algobase.logic.parser.CliSyntax.PREFIX_FORMAT;
+import static seedu.algobase.logic.parser.CliSyntax.PREFIX_PATH;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import seedu.algobase.commons.exceptions.DataConversionException;
+import seedu.algobase.commons.exceptions.IllegalValueException;
+import seedu.algobase.commons.util.FileUtil.Format;
+import seedu.algobase.commons.util.JsonUtil;
+import seedu.algobase.logic.commands.exceptions.CommandException;
+import seedu.algobase.model.Model;
+import seedu.algobase.storage.JsonSerializableAlgoBase;
+
+/**
+ * Imports AlgoBase from specified location.
```

Comment:
Is it true that by importing a save file AlgoBase will essentially overwrite the save file at its default location?
Is it better if we change the default save file path at the same time so that the old save file won't be overwritten?